packages/math-input/src/keypad/index.jsx

```diff
@@ -59,6 +60,9 @@ const LatexButton = withStyles(theme => ({
           fontSize: '80%',
           right: 'calc(-13%)',
           top: '-0.25em'
+        },
+        '&.mq-empty': {
+          minHeight: '1.4em'
```

Comment:
I see we're using em here. Do we set the fontSize of the parent element (do we have control on it?)?